contracts/package.json

```diff
@@ -17,8 +15,8 @@
     "@nomiclabs/hardhat-truffle5": "^2.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
     "@nomiclabs/hardhat-web3": "^2.0.0",
-    "@openzeppelin/contracts": "^3.2.0",
     "@openzeppelin/test-helpers": "^0.5.10",
+    "@truffle/debug-utils": "^4.1.1",
```

Comment:
Just curious what this package is for?

Reply:
I'm not sure, we already had it. I just moved it from dependencies to dev dependencies.